src/pages/@owner/food-truck-form/hooks/use-food-truck-form.ts

```
@@ -0,0 +1,165 @@
+
+    // Sub-hooks
+    useBasicInfo,
+    useTime,
+    useCategories,
+    useMenuInfo,
+    useRegion,
+  };
+};
+
+export const createFoodTruckFormMethods = (initialData?: FoodTruckFormData) => {
+  return useForm<FoodTruckFormData>({
```

ESLint

[error] React Hook "useForm" is called in function "createFoodTruckFormMethods" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)